src/metabase/models/parameter_card.clj

```diff
@@ -3,19 +3,21 @@
    [metabase.models.interface :as mi]
    [metabase.util :as u]
    [metabase.util.i18n :refer [tru]]
+   [metabase.util.schema :as su]
+   [schema.core :as s]
    [toucan.db :as db]
    [toucan.models :as models]))
 
 (models/defmodel ParameterCard :parameter_card)
 
 (defonce ^{:doc "Set of valid parameterized_object_type for a ParameterCard"}
```

Comment:
just randomly but why is this a `defonce`?

Reply:
just thought it gives a bit of performance to define constants with defonce. but I guess it's not dev friendly

Reply:
no performance aspect. just doesn't reload

```clojure
(defmacro defonce
  "defs name to have the root value of the expr iff the named var has no root value,
  else expr is unevaluated"
  {:added "1.0"}
  [name expr]
  `(let [v# (def ~name)]
     (when-not (.hasRoot v#)
       (def ~name ~expr))))
```

So the only thing that it ultimately does is `(def thing val)` with a conditional to not do that if it already exists.